templates/webpack.angular.js

```diff
@@ -74,12 +56,71 @@ module.exports = env => {
             "setImmediate": false,
             "fs": "empty",
         },
-        module: { rules },
-        plugins,
+        module: {
+            rules: [
+                { test: /\.html$|\.xml$/, use: "raw-loader" },
+                // Root stylesheet gets extracted with bundled dependencies
+                { test: new RegExp(mainSheet), use: "css-loader?url=false" },
+                // Other CSS files get bundled using the raw loader
+                { test: /\.css$/, exclude: new RegExp(mainSheet), use: "raw-loader" },
+                // SASS support
+                { test: /\.scss$/, use: ["raw-loader", "resolve-url-loader", "sass-loader"] },
+                // Compile TypeScript files with ahead-of-time compiler.
+                { test: /.ts$/, loader: "@ngtools/webpack" },
+            ],
+        },
+        plugins: [
+            // Vendor libs go to the vendor.js chunk
+            new webpack.optimize.CommonsChunkPlugin({
+                name: ["vendor"],
+            }),
+            // Define useful constants like TNS_WEBPACK
+            new webpack.DefinePlugin({
+                "global.TNS_WEBPACK": "true",
+            }),
+            // Copy assets to out dir. Add your own globs as needed.
+            new CopyWebpackPlugin([
+                { from: "css/**" },
```

Comment:
Let's copy just for sure...
Ultimately all CSS should either be loaded through css-loader from the {N} framework or the AngularCompilerPlugin should load them (raw-loader?) and compile them.